pkg/chain/bitcoin/handle.go

```diff
@@ -0,0 +1,8 @@
+package bitcoin
+
+// Handle serves as an interface abstraction around bitcoin network queries
+type Handle interface {
+	Broadcast(transaction string) error
+	VbyteFee() (int32, error)
```

Comment:
We may need to either make this more generic (because it could return several different fees) or make the name more explicit (because it's checking the suggested fee for a particular target confirmation time).

Reply:
Went with the more specific name. If we ever find that we want fees from blocks other than 25, it's easy to abstract the code at that time